lib/ControlledRefreshableListView.js

```diff
@@ -37,7 +38,16 @@ var ControlledRefreshableListView = React.createClass({
   handleScroll(e) {
     var scrollY = e.nativeEvent.contentInset.top + e.nativeEvent.contentOffset.y
 
-    if (!(this.props.ignoreInertialScroll && this.isTouching)) {
+    if (scrollY == 0 && this.scrollingUp && !this.isTouching) {
```

Comment:
Lines 41-48 tells me when the user stops pulling and the scroll goes back to 0. Only then I set isRefreshing to false. Lines 46-48 were necesary because if I update the list while user is pulling, scroll position is set to 0 and then gets back to last position.